bin/casher

```diff
@@ -72,10 +73,14 @@ class Casher
       end
     end
     unless archive_found
-      msg "could not download cache", :red
-      if File.exist? @fetch_tar
-        rm @fetch_tar
-      end
+      cleanup_cache
```

Comment:
Given how small the body of the `unless` block is now, I would switch this to a trailing condition a la `cleanup_cache unless archive_found`.